compiler/plugins/input/StableHLO/stablehlo-iree/Conversion/Passes.td

```diff
@@ -51,6 +51,11 @@ def LegalizeChlo :
   let summary = "Legalizes from CHLO ops flow to StableHLO and Shape ops";
 }
 
+def LegalizeStableHLOCustomCalls :
+    Pass<"iree-stablehlo-legalize-custom-calls", "func::FuncOp"> {
+  let summary = "Inlines codegenerated solvers for StableHLO ops";
```

Comment:
Is this specific to solvers? (I mean nit, some of these as factorizations and the factored form is used in solvers ...)

Reply:
I rephrased away from `solver`. Originally I intended to focus on solvers but am inching away. Technically `triangular_solve` and `cholesky` are individual ops but we can argue they are promoted custom ops.